src/main/java/io/cryostat/net/web/http/api/v1/TargetRecordingsPostHandler.java

```diff
@@ -201,6 +201,14 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
                                                     attrs.get("metadata"),
                                                     new TypeToken<Metadata>() {}.getType());
                                 }
+                                boolean archiveOnStop = false;
+                                if (attrs.contains("archiveOnStop")) {
+                                    Pattern bool = Pattern.compile("true|false");
```

Comment:
`Pattern` seems like overkill here :-)

`attr.equals("true") || attr.equals("false")` would do about the same, right?

Reply:
The `Patter` is being used in the `toDisk` `if` too, so I will replace it in both `if`s.